client/components/theme/index.jsx

```diff
@@ -61,11 +61,44 @@ const Theme = React.createClass( {
 		actionLabel: React.PropTypes.string
 	},
 
+	areButtonContentsEqual( nextButtonContents, buttonContents ) {
+		if ( ! isEqual( Object.keys( nextButtonContents ), Object.keys( buttonContents ) ) ) {
+			return false;
+		}
+
+		return every( buttonContents, ( option, name ) => {
```

Comment:
Returning `true` here instead of this `every` call makes no difference to the results for me.